src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,151 @@
+package duke;
+
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+
+/**
+ * Handles loading and saving of tasks to a file.
+ */
+public class Storage {
```

Comment:
This implementation seems redundant. If FILE_PATH is already a static constant, there is no need to have this.filePath = FILE_PATH. You can remove the entire constructor and only have static methods in Storage since there isn't any reason why we need several instances of Storage